test/suite/test_oligarch06.py

```diff
@@ -63,7 +66,8 @@ def test_oligarch06(self):
         # Sometimes there are data corruption bugs - apparently we act for an evicted
         # page back, and get one with the wrong checksum.  When that doesn't happen,
         # sometimes all items written to the leader have not gotten to the follower.
-        self.skipTest('fails due to data corruption')
+        if self.nitems > 1000:
+            self.skipTest('fails due to data corruption')
```

Comment:
Please try again with the latest PALM fix in SLS-485, we should be able to remove the skip with 100000 nitems.